src/wallet/rpcwallet.cpp

```diff
@@ -3788,6 +3788,81 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
     return ret;
 }
 
+UniValue sethdseed(const JSONRPCRequest& request)
+{
+    CWallet* const pwallet = GetWalletForJSONRPCRequest(request);
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() > 2) {
```

Comment:
Could have a test for extra parameters. See [example](https://github.com/bitcoin/bitcoin/pull/12083/files#diff-695d736c7d2a38a42ae900546aed52f8).

Reply:
Done